src/Filter.js

```diff
@@ -25,7 +25,7 @@ class Filter {
       if (this.filterTypes.has(key)) {
         this.filter.type = this.filterTypes.get(key)
       } else {
-        console.log(`Filter key ${key} not recognized`)
+        throw Error(`Filter key ${key} not recognized`)
```

Comment:
Line 26 can return a similar Error: (Chrome 60)
```
The provided value 'asdf' is not a valid enum value of type OscillatorType.
```
May still be a good idea to keep this though.